docs/orchestration/agents/kubernetes.md

```diff
@@ -163,6 +163,10 @@ prefect-agent-845798bb59-s7wxg   1/1     Running   0          5s
 
 You are now ready to run some flows!
 
+#### Permissions
+
+In AWS deployments using Amazon Elastic Kubernetes Service, S3 is not accessible by default to Amazon EKS. To enable S3 access by your kubernetes cluster on EKS, add the necessary permissions (AmazonS3FullAccess or AmazonS3ReadOnlyAccess) directly to the NodeInstanceRole used by aws-auth-cm.yaml after launching worker nodes and before applying aws-auth-cm.yaml with kubectl.
```

Comment:
Can you add something in here to qualify that S3 isn't required? Something like "If you need S3 access, note that..." or something else. Not all users will be affected by this.